extensions/geode-modules-tomcat8/src/distributedTest/java/org/apache/geode/modules/session/EmbeddedTomcat8.java

```diff
@@ -86,19 +58,18 @@ public EmbeddedTomcat8(String contextPath, int port, String jvmRoute)
     // Otherwise we get NPE when instantiating servlets
     rootContext.setIgnoreAnnotations(true);
 
-
     AuthConfigFactory factory = new AuthConfigFactoryImpl();
     new SimpleAuthConfigProvider(null, factory);
     AuthConfigFactory.setFactory(factory);
 
     // create engine
-    engine = container.getEngine();
+    Engine engine = container.getEngine();
     engine.setName("localEngine");
     engine.setJvmRoute(jvmRoute);
 
     // create http connector
     container.setPort(port);
-    Connector httpConnector = container.getConnector();// ((InetAddress) null, port, false);
+    container.getConnector();// ((InetAddress) null, port, false);
```

Comment:
Does this getter still have side effects if we don't assign it to anything?  If not, can we just delete this line?

Reply:
:+1: